iocore/net/quic/QUICFrame.h

```diff
@@ -234,7 +234,7 @@ class QUICAckFrame : public QUICFrame
     size_t size() const;
     Ptr<IOBufferBlock> to_io_buffer_block(size_t limit) const;
     uint64_t first_ack_block() const;
-    void add_ack_block(const AckBlock block);
+    void add_ack_block(const AckBlock &block);
```

Comment:
Nitpick:  AckBlock is 16 bytes in size, looks to be small enough to be passed in registers ( https://godbolt.org/z/MdrabW ).  So this is a small deoptimization.

Reply:
Was this change made by hand or some tools?
While I want to make Ack processing fast as possible as we can, I also want to take the advantage of auto-correct (if it was done by tools). I guess it beats the unfortunate case that we accidentally copy big objects.

Reply:
Yes you can make an argument that, except in rare cases, it's overoptimizing to pass class types by value.  It's future safe if new member variables get added to the class.